mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/utbetaling/UtbetalingService.kt

```diff
@@ -381,7 +384,20 @@ class UtbetalingService(
         )
     }
 
-    private fun getDeltakelser(
+    private fun resolveStengtHosArrangor(
+        periode: Periode,
+        stengt: List<GjennomforingDto.StengtPeriode>,
+    ): Set<StengtPeriode> {
+        return stengt
+            .mapNotNull { stengt ->
+                Periode(stengt.start, stengt.slutt.plusDays(1)).intersect(periode)?.let {
+                    StengtPeriode(it.start, it.slutt, stengt.beskrivelse)
```

Comment:
Synes det er litt forvirrende at StengtPeriode her har ikke inclusive sluttDato akkurat her, men ellers så har den det? Det ser ut som det blir riktig fordi den mappes tilbake til periode med Periode.of og ikke Periode.fromInclusiveDates senere, men når vi har start og slutt separert ellers så burde man jo mappe med den andre. Og denne stengtPeriode tolkes sånn ellers i koden 🤔